packages/salesforcedx-vscode-apex/src/embeddedSoql/soqlCompletion.ts

```diff
@@ -84,6 +84,7 @@ function getSOQLVirtualContent(
 }
 
 export const soqlMiddleware: Middleware = {
+  // @ts-ignore
```

Comment:
we should add a comment with any ts-ignore to say why we're ignoring out safety net